plugins/CustomVariables/Reports/GetCustomVariables.php

```diff
@@ -34,5 +35,40 @@ public function configureView(ViewDataTable $view)
         $view->config->addTranslation('label', Piwik::translate('CustomVariables_ColumnCustomVariableName'));
         $view->requestConfig->filter_sort_column = 'nb_actions';
         $view->requestConfig->filter_sort_order  = 'desc';
+
+        $view->config->filters[] = function (DataTable $table) use ($view) {
+            if($this->isReportContainsUnsetVisitsColumns($table)) {
```

Comment:
This fails on PHP 5.3 see eg http://builds-artifacts.piwik.org/piwik/piwik/9649/17844/PivotByDimension_pivoted_columns_report . Instead we need to use `use ($view, $self)`